components/UseCases.js

```diff
@@ -155,8 +158,8 @@ export default function UseCases() {
             parallel cron jobs that don't require a separate batch processing framework.
           </p>
           <p className="mb-3">
-            Temporal support for long-running activities and unlimited retries also makes it a great
-            fit for monitoring use cases.
+            Temporal's support for long-running activities and unlimited retries also makes it a
```

Comment:
```suggestion
            Temporal's support for long-running Activities and unlimited retries also makes it a
```

In this case we want to capitalize Activities.

I may have missed some above, but basically anytime we refer to a Workflow and Activity as a concept/component of the Temporal system - capitalize it.
That way users can differentiate between generic "workflow" terminology and the Temporal system.